Gems/PhysX/Code/Source/Utils.cpp

```diff
@@ -1177,17 +1177,11 @@ namespace PhysX
             }
         }
 
-        AZ::Vector3 GetTransformScale(AZ::EntityId entityId)
+        float GetTransformScale(AZ::EntityId entityId)
```

Comment:
Shouldn't this function be `GetUniformTransformScale` now to be clear?

Reply:
Transform scale can't be anything but uniform.